analysis/civic/civic_harvester_example.py

```diff
@@ -0,0 +1,68 @@
+"""Create an example json file for CIViC Harvester."""
```

Comment:
A README.md in this directory would be useful.  At first read, I thought it was creating test fixtures or notebook setup data.